karols-frontend/src/stories/index.js

```diff
@@ -2225,3 +2226,8 @@ const weekTimeSlots = [
 // storiesOf("ResultCalendar", module).add("Render the calendar with time", () => (
 //   <ResultCalendar />
 // ))
```

Comment:
test also with a CardModel with the appropriate props, the console.log should be here